src/Robo/Plugin/Commands/DevelopmentModeBaseCommands.php

```diff
@@ -303,14 +271,14 @@ protected function drushDeployLando(string $siteDir = 'default'): Result
         }
         return $this->taskExecStack()
             ->dir("$this->drupalRoot/sites/$siteDir")
-            ->exec("lando drush deploy --yes")
+            ->exec("$localEnvironmentType->value drush @$siteDir.$localEnvironmentType->value deploy --yes")
             // Import the latest configuration again. This includes the latest
             // configuration_split configuration. Importing this twice ensures that
             // the latter command enables and disables modules based upon the most up
             // to date configuration. Additional information and discussion can be
             // found here:
             // https://github.com/drush-ops/drush/issues/2449#issuecomment-708655673
-            ->exec("lando drush config:import --yes")
+            ->exec("$localEnvironmentType->value drush @$siteDir.$localEnvironmentType->value config:import --yes")
```

Comment:
How does this work if the local environment runs `drush` natively`? Does the ` ` space before `drush` cause an issue?